docker/sysbox/README.md

```diff
@@ -0,0 +1,31 @@
+# Preparing your code base for Remote Execution
+
+This is an example project that validates Sysbox setup for your EngFlow cluster
+works correctly.
+
+Use as companion to https://docs.engflow.com/re/client/sysbox.html
+
+Usage:
+
+- Follow instructions in https://docs.engflow.com/re/client/sysbox.html to generate
```

Comment:
I think this creates a bit of confusion. Sysbox != DinD. Please differentiate the 2 concepts more.

Reply:
Modified the text to clarify a bit more. Let me know if this looks good.